.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea
+.vscode
```

Comment:
I think these should go in your personal .gitignore config and not each project. Otherwise we'd end up with every developers editor ever which could be a large ignore list.

Reply:
fixed